### ISSUE TITLE
Breadcrumb should be navigable with keyboard

### ISSUE THREAD

**Issue by just-be-dev** [oxidecomputer/console#471]
You can tab into breadcrumbs but you can't navigate further from there. After initially tabbing into the component a user should be able to use arrow keys to further navigate to individual items.

See the aria recommendations for [keyboard navigation inside components](https://www.w3.org/TR/wai-aria-practices-1.1/#kbd_general_within) for general implementation guidance.

## Resources
- https://www.w3.org/TR/wai-aria-practices/examples/breadcrumb/index.html
- https://www.aditus.io/aria/aria-current/
